packages/cardpay-sdk/package.json

```diff
@@ -32,14 +39,15 @@
     "web3-utils": "1.5.2"
   },
   "devDependencies": {
+    "tsup": "^5.11.7",
     "typescript": "4.2.3"
   },
   "files": [
     "dist"
   ],
   "scripts": {
-    "postinstall": "node bin/codegen.js",
-    "prepack": "tsc"
+    "postinstall": "if [ -f ./bin/codegen.js ]; then node bin/codegen.js; fi",
```

Comment:
@lukemelia this is an interesting gotcha--the package in npm doesn't actually include `bin/`. this script only works for maintainers of this source code, not for pkgs downloaded from npm. This actually blows up if you yarn install this package because of this file not being present